client/components/FAQ.scss

```diff
@@ -0,0 +1,118 @@
+.c_header{
+    margin-top:84px;
+    width: 607px;
+  height: 56px;
+  font-family: Roboto;
+  font-size: 48px;
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.17;
+  letter-spacing: normal;
+  color: #01131f;
+  margin-left:64px;
+  margin-right:769px;
+}
+.c_rectangle_general_information{
+    margin-top:80px;
+    width: 862px;
+    height: 640px;
+    margin-left:64px;
+    padding-left:32px;
+    border-radius: 8px;
+    box-shadow: 0 0 16px 0 rgba(0, 0, 0, 0.16);
+    background-color: #fdfeff;
+    padding-top:40px;
+    margin-bottom:80px;
+}
+.c_gen_information{
+    text-align:left;
+    margin-bottom:24px;
+    width: 288px;
+    height: 40px;
+    font-family: Roboto;
+    font-size: 32px;
+    font-weight: bold;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: 1.25;
+    letter-spacing: normal;
+    color: var(--black);
```

Comment:
I don't think the `--black` var is actually defined anywhere, so this will probably do nothing. I think most of these styles should be covered by more global styles; try removing all the styles, and add back only the ones that are missing.